spec/cypress/support/commands.js

```diff
@@ -14,6 +14,10 @@ Cypress.Commands.add("logout", () => {
   cy.location("pathname").should("eq", "/");
 });
 
+Cypress.Commands.add("clickCommitButton", () => {
+  cy.get("[name=commit]").click();
+});
```

Comment:
In my opinion it would be better if this were added directly into the step definition if it isn't already used anywhere else